components/Api/ApiChanges/index.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import ApiChanges from './index';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+type Story = StoryObj<typeof ApiChanges>;
+type Meta = MetaObj<typeof ApiChanges>;
+
+const version = 'v10.0.0';
+export const Default: Story = {
+  args: {
+    update: {
+      type: 'added',
+      version: [version],
+    },
+    changes: [
+      {
+        version: version,
+        'pr-url': 'https://test.com',
```

Comment:
to have perfect mock
```suggestion
        'pr-url': 'https://github.com/nodejs/node/pulls',
```